tests/time_tests/test_runner/test_timetest.py

```diff
@@ -40,3 +40,5 @@ def test_timetest(instance, executable, niter):
     }
     retcode, aggr_stats = run_timetest(exe_args, log=logging)
```

Comment:
add Path().resolve() for `executable` and `model` from `exe_args`

Reply:
It's already done here - https://github.com/openvinotoolkit/openvino/blob/a5c01343c1bde67b7acbc74243660f0e2367afe4/tests/time_tests/scripts/run_timetest.py#L76